third_party/externs/ts/gapi/drive.d.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+declare namespace gapi.client {
+  export module drive {
```

Comment:
Unfortunately not. The only place I found that hosts declaration files for these is [this repo](https://github.com/Bolisov/typings-gapi/tree/master/) which looks like it's maintained (has drive v3, recently released), but it's not on npm, and instead uses `typings`, which is deprecated in favor of npm's `@types`.